main.py

```diff
@@ -14,18 +14,20 @@ def get(self, platform_name):
 
         self.redirect(platform.get_last_snapshot_url())
 
+class RevisionHandler(webapp.RequestHandler):
+    def get(self, platform_name):
+        platform = get_platform_string(platform_name, self.request)
+        last_revision = get_revision(platform)
+        self.response.out.write(last_revision)
```

Comment:
Case where `last_revision` is not retrieved is not handled there. 
What do you think of something like http://pastebin.com/i0605kq7?